src/cinder/PolyLine.cpp

```diff
@@ -71,6 +71,12 @@ void PolyLineT<T>::offset( const T &offsetBy )
 }
 
 template<typename T>
+void PolyLineT<T>::reverse()
+{
+    std::reverse( mPoints.begin(), mPoints.end() );
```

Comment:
My preference would be to return a copy but this matches what the `scale` and `offset` methods do.